src/client/sandbox/xhr.js

```diff
@@ -95,7 +95,7 @@ export default class XhrSandbox extends SandboxBase {
         xmlHttpRequestProto.send = function () {
             var xhr = this;
 
-            xhrSandbox.emit(xhrSandbox.XHR_SEND_EVENT, { xhr });
+            xhrSandbox.emit(xhrSandbox.XHR_SENT_EVENT, { xhr });
```

Comment:
Yes, you are right. 
I propose to rename event as `XHR_BEFORE_SEND_EVENT`. 
/cc @LavrovArtem @churkin